packages/smithy-client/src/parse-utils.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Give an input string, strictly parses a boolean value.
+ *
+ * @param value The boolean string to parse.
+ * @returns true for "true", false for "false", otherwise an error is thrown.
+ */
+export const parseBoolean = (value: "true" | "false"): boolean => {
```

Comment:
Allow passing string in parseBoolean, as it throws error when it's value is neither true/false.
```suggestion
export const parseBoolean = (value: string): boolean => {
```